libs/openFrameworks/3d/ofNode.cpp

```diff
@@ -231,6 +231,19 @@ void ofNode::rotateAround(float degrees, const ofVec3f& axis, const ofVec3f& poi
 }
 
 //----------------------------------------
+void ofNode::lookAt(const ofVec3f& lookAtPosition){
+    auto relPosition = (getGlobalPosition() - lookAtPosition);
+    auto radius = relPosition.length();
+    if(radius>0){
+        auto latitude = ofRadToDeg(acos(relPosition.y / radius)) - 90;
+        auto longitude = ofRadToDeg(atan2(relPosition.x , relPosition.z));
```

Comment:
Interesting! 

One c++ reference says that in that case a `domain error` (<http://www.cplusplus.com/reference/cmath/atan2/>) will be raised, but this reference (http://en.cppreference.com/w/cpp/numeric/math/atan2) says a domain error will not occur "If the implementation supports IEEE floating-point arithmetic," so we should be safe for floats, doubles, and long doubles =) 

And when i try atan2(0.f,0.f) myself i get 0, at least in clang, as you reported. `0`, anyway, is what we might want for longitude in that case. 

+1